circle.yml

```diff
@@ -1,11 +1,16 @@
 test:
-  override:
-    - chmod +x yamlvalidator
-    - ./yamlvalidator
-    - chmod +x ensurefilenames.sh
-    - ./ensurefilenames.sh
+  pre:
+    # install python client
+    - pip install git+https://github.com/demisto/demisto-py.git@test-content
+
+    # validate yaml
+    - chmod +x ./Tests/yamlvalidator
+    - ./Tests/yamlvalidator
```

Comment:
did you add somewhere to check that each integration/script has a description here?